benchmark/_cli.js

```diff
@@ -125,3 +125,24 @@ CLI.prototype.shouldSkip = function(scripts) {
 
   return skip;
 };
+
+/**
+ * Extracts the CPU core setting from the CLI arguments.
+ * @returns {string|null} The CPU core setting if found, otherwise null.
```

Comment:
```suggestion
 * @returns {string} The CPU core setting if found, otherwise null.
```

Reply:
no it do return null when there is no CPUSET variable, that's how we identify wether to use spawn or fork